apps/fabric-website/src/pages/Controls/MessageBarPage/MessageBarPage.tsx

```diff
@@ -1,7 +1,37 @@
 import * as React from 'react';
+import { IPageSectionProps, Markdown } from '@uifabric/example-app-base/lib/index2';
 import { ControlsAreaPage, IControlsPageProps } from '../ControlsAreaPage';
 import { MessageBarPageProps } from './MessageBarPage.doc';
+import { Platforms } from '../../../interfaces/Platforms';
+
+const baseUrl =
+  'https://github.com/microsoft/fluentui/tree/master/apps/fabric-website/src/pages/Controls/MessageBarPage/';
 
 export const MessageBarPage: React.FunctionComponent<IControlsPageProps> = props => {
-  return <ControlsAreaPage {...props} {...MessageBarPageProps[props.platform]} />;
+  return (
+    <ControlsAreaPage
+      {...props}
+      {...MessageBarPageProps[props.platform]}
+      otherSections={_otherSections(props.platform) as IPageSectionProps[]}
+    />
+  );
 };
+
+function _otherSections(platform?: Platforms): IPageSectionProps<Platforms>[] | undefined {
+  switch (platform) {
+    case 'ios':
+      return [
+        {
+          sectionName: 'Implementation',
+          editUrl: baseUrl + 'docs/ios/MessageBarImplementation.md',
+          content: (
+            <Markdown>
```

Comment:
Markdown wrapper isn't needed here--if you use a string as content it will automatically be rendered as markdown. (same in other instances)